cyences_app_for_splunk/default/macros.conf

```diff
@@ -1016,6 +1016,12 @@ definition = 10
 definition = ("cyences.com")
 # Add the comma separated values inside the brackets.
 
+# Team Email Configuration
+[cs_soc_team_email]
+definition = 
+
+[cs_compliance_team_email]
+definition = 
```

Comment:
Are we handling what to do when this macro is empty?

Reply:
Yes, in case of empty definition, Compliance alerts will go to only default receipients if it's configured otherwise no one get an alert.